shell/common/skp_shader_warmup_unittests.cc

```diff
@@ -19,15 +19,19 @@
 #include "flutter/shell/common/switches.h"
 #include "flutter/shell/version/version.h"
 #include "flutter/testing/testing.h"
+#include "include/core/SkFont.h"
 #include "include/core/SkPicture.h"
 #include "include/core/SkPictureRecorder.h"
 #include "include/core/SkSerialProcs.h"
+#include "include/core/SkTextBlob.h"
+
+#if defined(OS_FUCHSIA)
+#include "lib/sys/cpp/component_context.h"
+#include "third_party/skia/include/ports/SkFontMgr_fuchsia.h"
 
```

Comment:
You need an #endif here or the namespace declarations are stripped out, this will cause problems on non-fuchsia builds

Reply:
this ifdef block wraps basically the entire file including the close braces on the namespace. I'll take an action item to comment the endif with "// defined(OS_FUCHSIA)" though